packages/mui-base/src/Portal/Portal.test.tsx

```diff
@@ -129,9 +129,9 @@ describe('<Portal />', () => {
   });
 
   it('should change container on prop change', () => {
-    function ContainerTest(props) {
+    function ContainerTest(props: { containerElement?: false | undefined; disablePortal?: true | undefined; }) {
```

Comment:
That's more life-like:

```suggestion
    function ContainerTest(props: { containerElement?: boolean; disablePortal?: boolean; }) {
```